### ISSUE TITLE
[FeatureRequest] `geom_density_ridges` for bounded data

### ISSUE THREAD

**Issue by wbvguo** [wilkelab/ggridges#85]
Hi,

Thanks for maintaining this tool. Would it be possible to have an option for setting bounds when plotting the density ridge plot?

Just as the `geom_density` did in this github post:

https://github.com/tidyverse/ggplot2/issues/3387

Thanks!

**Comment by rimonim** [comment]
I agree that this would be a nice feature. Until it gets implemented, I'll note that the effect can be pretty easily achieved by setting stat to "density" (remember to add `height = ..density..`)

For example:
`
ggplot(diamonds, aes(x = price, y = cut, fill = cut, height = ..density..)) +
  geom_density_ridges(stat = "density", bounds = c(0, 20000))
`
